odo/backends/sql.py

```diff
@@ -103,7 +103,7 @@
     mssql.UNIQUEIDENTIFIER: string,
     # The SQL Server TIMESTAMP value doesn't correspond to the ISO Standard
     # It is instead just a binary(8) value with no relation to dates or times
-    mssql.TIMESTAMP: bytes_,
+    # mssql.TIMESTAMP: bytes_,
```

Comment:
Yes... on a (somewhat) I think I was the one who added `BIT`-> `datashape.bool_` to `revtypes` because MSSQL allows you to have null `BIT`. (My database architect loves `BIT`, so this was a problem for me.) I had to add that line to prevent a error because otherwise `BIT` didn't have anything to convert into so blaze would just crap out on encountering a nullable `BIT`. 

Part of me wonders if we need the blaze ecosystem realize what dialect the data is coming from and look up the appropriate conversions that way.

Thanks for keeping this thing alive. I use blaze frequently. I feel bad I can't thank continuum more for such an excellent tool.